### ISSUE TITLE
Bcrypt and blowfish

### ISSUE THREAD

**Issue by Ekultek** [Ekultek/Dagon#26]
```
C:\Users\thomas_j_perkins\Documents\bin\python\dagon>python dagon.py -c 098f6bcd4621d373cade4e832627b4f6 --bruteforce -A 500
Dagon .. Advanced Hash Manipulation v1.9.17.29(dev)
Clone: https://github.com/ekultek/dagon.git

[16:34:58 INFO] Checking program integrity...
[*] Starting up at: 1499031305.06..

[16:35:05 INFO] Using algorithm, BLOWFISH..
No handlers could be found for logger "passlib.handlers.bcrypt"
[16:35:07 CRITICAL] User exited process...
```

**Comment by Ekultek** [comment]
Looking into this

**Comment by Ekultek** [comment]
Created issue on `passlib` repo here: https://bitbucket.org/ecollins/passlib/issues/93/no-handlers-could-be-found-for-logger

**Comment by Ekultek** [comment]
This is fixed via d07c48a waiting on merge of pull request

**Comment by Ekultek** [comment]
Fixed via cdf85e3
